stack-npm/pom.xml

```diff
@@ -0,0 +1,261 @@
+              <execution>
+                <id>unzip-vertx-stack-full</id>
+                <phase>compile</phase>
+                <goals>
+                  <goal>unpack-dependencies</goal>
+                </goals>
+                <configuration>
+                  <includeArtifactIds>vertx-stack-dist</includeArtifactIds>
+                  <includeClassifiers>full</includeClassifiers>
+                  <outputDirectory>${project.build.directory}/npm/full</outputDirectory>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <!--
+            Copy a filtered version of the package.json file
+            -->
+            <artifactId>maven-resources-plugin</artifactId>
+            <version>2.7</version>
+            <executions>
+              <execution>
+                <id>prepare-npm-min</id>
+                <phase>validate</phase>
+                <goals>
+                  <goal>copy-resources</goal>
+                </goals>
+                <configuration>
+                  <outputDirectory>${project.build.directory}/npm/min/vert.x-${project.version}</outputDirectory>
+                  <resources>
+                    <resource>
+                      <directory>src/main/npm/min</directory>
+                      <filtering>true</filtering>
+                    </resource>
+                  </resources>
+                </configuration>
+              </execution>
+              <execution>
+                <id>prepare-npm-base</id>
+                <phase>validate</phase>
+                <goals>
+                  <goal>copy-resources</goal>
+                </goals>
+                <configuration>
+                  <outputDirectory>${project.build.directory}/npm/base/vert.x-${project.version}</outputDirectory>
+                  <resources>
+                    <resource>
+                      <directory>src/main/npm/base</directory>
+                      <filtering>true</filtering>
+                    </resource>
+                  </resources>
+                </configuration>
+              </execution>
+              <execution>
+                <id>prepare-npm-full</id>
+                <phase>validate</phase>
+                <goals>
+                  <goal>copy-resources</goal>
+                </goals>
+                <configuration>
+                  <outputDirectory>${project.build.directory}/npm/full/vert.x-${project.version}</outputDirectory>
+                  <resources>
+                    <resource>
+                      <directory>src/main/npm/full</directory>
+                      <filtering>true</filtering>
+                    </resource>
+                  </resources>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <artifactId>exec-maven-plugin</artifactId>
+            <groupId>org.codehaus.mojo</groupId>
```

Comment:
We should use the maven frontend plugin to run npm. 
See https://github.com/vert-x3/vertx-web/commit/a999841ff1b66cee59456576e3bbc957c1ebc6b1#diff-600376dffeb79835ede4a0b285078036R200